plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -354,7 +365,11 @@ public static LongReadFunction dateReadFunction()
         };
     }
 
-    public static LongWriteFunction dateWriteFunction()
+    /**
+     * If driver supports {@link LocalDate}, use {@link #dateWriteFunctionUsingLocalDate} instead.
```

Comment:
Use `@deprecated` tag for such JavaDoc, shows up well in the IDE (see https://docs.oracle.com/javase/7/docs/technotes/guides/javadoc/deprecation/deprecation.html for full example).